db/schema.rb

```diff
@@ -235,16 +235,6 @@
     t.index ["user_id"], name: "fk_rails_5b1168410c"
   end
 
-  create_table "mentor_profiles", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci", force: :cascade do |t|
```

Comment:
@iHiD This schema file was produced after rebasing. Is this correct? I looked through the models and `MentorProfile` doesn't exist anymore, but on master, the corresponding table still exists in the schema.

Reply:
Good catch.

`mentor_profiles` is from an old branch that never get deployed. So that table is still in my local db, but not in production. I should take a new production dump and work from that at some point.